component_tabs.yaml

```diff
@@ -1,3 +1,63 @@
 patterns/accordion:
   accessibility: True
   design_guidelines: False
+patterns/breadcrumbs:
+  accessibility: True
+  design_guidelines: False
```

Comment:
Nit picking a bit - but could it simply default to false? so if it's not there in config, it means there is no tab for it?

Reply:
Turns out it was already working that way. I removed all the lines that init the value to False.